packages/lava-queue/src/queue.ts

```diff
@@ -106,8 +106,8 @@ export class Queue {
     this.lavaPlayer.pause(false);
   }
 
-  search(text: string): Promise<TrackResponse> {
-    return this.player.node.load(`ytsearch:${text}`);
+  search(text: string, searchEngine = "ytsearch"): Promise<TrackResponse> {
```

Comment:
Please make this an enum, or some sort of configurable prop that can detect engines. it would be nice rather than just a free string of any value

Reply:
If you do not want to add a plugin discovery engine, and want it to be a arbitrary string, please add error handling in case the search Engine is not found